src/irmin-pack/checks.ml

```diff
@@ -312,6 +312,71 @@ module Make (M : Maker) = struct
         (term_internal $ setup_log, info ~doc "integrity-check-inodes")
   end
 
+  module Stats_commit = struct
+    let conf root = Conf.init ~readonly:true ~fresh:false root
+
+    let commit =
+      let open Cmdliner.Arg in
+      value
+      & opt (some string) None
+      & info [ "commit" ] ~doc:"The commit whose underlying tree is traversed."
+          ~docv:"COMMIT"
+
+    let all_paths =
```

Comment:
`all_paths` doesn't reflect well the fact that the value should contain the path to a destination file. 

What about `--blob_paths_dump_destination path` or `--destination_of_paths_dump path` or `--dump_blob_paths_to path` or `--write_blob_paths_to path`?

Reply:
Thanks for the suggestions, I went with `dump_blob_paths_to`.